src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(/[;:]/)
+    .map((trimmedString) => trimmedString.trim())
+    .filter((nonEmptyString) => nonEmptyString.length > 0)
+    .reduce((resultObject, currentString, stringIndex, allStrings) => {
+      if (stringIndex % 2 === 0) {
+        resultObject[currentString] = allStrings[stringIndex + 1];
+      }
+
+      return resultObject;
+    }, {});
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. You should align the chained methods to improve the code readability.